src/routes/apiDirectory.js

```diff
@@ -76,6 +77,7 @@ router.use('/goal-templates', goalTemplatesRouter);
 router.use('/events', eventRouter);
 router.use('/session-reports', sessionReportsRouter);
 router.use('/national-center', nationalCenterRouter);
+router.use('/email', emailRouter);
```

Comment:
Do we need to expose 2 routes? I think we are only using the one in ```/src/routes/admin/index.js``` and we could delete this one, which doesn't seem to check any permissions